### PR TITLE
finished parents UI profile

### PR DESCRIPTION
**Pull request by MaiCodes-exe**
the input form for the number of children needs to be fixed to be aligned with the other forms

### CODE REVIEW

File: src/front/styles/parents.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+#form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  font-size: medium;
+}
+
+#children {
+  margin-right: 20%;
```
Comment (vitortamberlini) [inline src/front/styles/parents.css:21]: I feel like this style *might* give you trouble when putting the label and the input side-by-side. I think adding padding is usually better, but that's just a suggestion 🤓

**Review by vitortamberlini — CHANGES_REQUESTED** [review]
Apart from the changes I requested in here, I also don't think you should pull these changes directly to `main`, as this is very dangerous in a real world situation.
Instead, you should do a pull request to an intermediary branch, like `frontend`.

I'll create a branch structure right away and post it over slack so we're all on the same page when opening PR's! But for now, just work on the things I mentioned here and by the time you finish, I'll have the image over our channel already!

File: src/front/js/layout.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { Home } from "./pages/home";
 import { Demo } from "./pages/demo";
 import { Single } from "./pages/single";
 import Login from "./pages/login";
+import Parents from "./pages/parentsview";
```
[the remark this answers is not in the document]
Comment (MaiCodes-exe) [reply src/front/js/layout.js:9]: I removed the "view" and renammed the page

File: src/front/js/pages/parentsview.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+const Parents = () => {
+  return (
+    <>
+      <div className="container-fluid">
+        <div className="container">
+          <img
+            src="https://previews.123rf.com/images/rosinka79/rosinka791704/rosinka79170400108/77286091-tambov-f%C3%A9d%C3%A9ration-de-russie-21-septembre-2016-portrait-de-la-famille-lego-figurines-d-un-p%C3%A8re-d-une-.jpg"
+            id="profile"
+          ></img>
+          <input id="bio"></input>
```
[the remark this answers is not in the document]
Comment (MaiCodes-exe) [reply src/front/js/pages/parentsview.js:13]: i meant an id tag so i could change it's size on using css

File: src/front/js/pages/parentsview.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            <div className="row mb-3">
+              <label
+                for="inputchildren"
+                className="col-sm-2 col-form-label"
+                id="children"
+              >
+                Number of Children
+              </label>
+              <select
+                class="form-select form-select-sm col-2"
```
[the remark this answers is not in the document]
Comment (MaiCodes-exe) [reply src/front/js/pages/parentsview.js:74]: you're right, i missed this one. The label thing is giving me trouble, even with css and boostrap i cant seem to make it work

File: src/front/js/layout.js [hunk 5]
```diff
@@ -32,6 +33,9 @@ const Layout = () => {
             <Route exact path="/login">
               <Login />
             </Route>
+            <Route exact path="/parentsview">
```
[the remark this answers is not in the document]
Comment (MaiCodes-exe) [reply src/front/js/layout.js:36]: fixed it

File: src/front/js/pages/parentsview.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+                <input
+                  type="password"
+                  className="form-control"
+                  id="password"
+                ></input>
+              </div>
+            </div>
+            <div className="row mb-3">
+              <label
+                for="inputchildren"
```
[the remark this answers is not in the document]
Comment (MaiCodes-exe) [reply src/front/js/pages/parentsview.js:67]: i removed that tag

File: src/front/styles/parents.css (hunk 1 above)
Comment (MaiCodes-exe) [reply src/front/styles/parents.css:21]: i'll ask the rest of group for a solution for this issue

File: src/front/js/pages/parentsview.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (vitortamberlini) [reply src/front/js/pages/parentsview.js]: No no... I meant the tag <input> itself... The way you put it there and by the name "bio" I'd assume you wanted an input w/ multiple lines, right?
What I meant is that the tag <textarea> does the job better in that scenario.
Check this link and see if it is what you wanted:
https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/textarea

**Review by vitortamberlini — COMMENTED** [review]
I commented over the previous commit about the tag <textarea>. Just check that and tell me if that is what you wanted.

If not, just reply here and I'll approve this commit 😎

File: src/front/js/pages/parentsview.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (emewarp) [reply src/front/js/pages/parentsview.js]: @MaiCodes-exe following on Vitor explanation:
-input is for 'short text'
-textarea is for longer text (a paragraph, email...)

so for a bio for example it makes sense to have a text area
for a name, email... it makes sense to have an input 

:)

**Review by ertip4geek — CHANGES_REQUESTED** [review]
Maise sorry last thing, for your pending PullRequestS, could you make 1 more commit without the file package-lock.json.
Because is causing 12 conflicts and is doing errors to be resolvev, just skip that file will be ok
thanks!:guiño:

File: src/front/js/pages/parents.js [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+        <Container>
+          <Row>
+            <Col className="col-2">
+              <img
+                src="https://previews.123rf.com/images/rosinka79/rosinka791704/rosinka79170400108/77286091-tambov-f%C3%A9d%C3%A9ration-de-russie-21-septembre-2016-portrait-de-la-famille-lego-figurines-d-un-p%C3%A8re-d-une-.jpg"
+                id="profile"
+              ></img>
+            </Col>
+            <Col className="col-5">
+              <input placeholder="Add your description" id="bio"></input>
```
Comment (vitortamberlini) [inline src/front/js/pages/parents.js:25]: I *still* think you didn't want to use this tag here, Maise. Why don't you try <textarea> and tell me what you think?
```suggestion
              <textarea placeholder="Add your description" id="bio" />
```

**Review by vitortamberlini — CHANGES_REQUESTED** [review]
Over the `parents.css`, just be careful with the names you are giving to the ids, okay?

I feel like it's better to be a little verbose now and not be confusing later! 🤓

**Pull request merged**